docs/specification.md

```diff
@@ -129,6 +129,7 @@ Full list:
 * [`balance_of`](#balance_of)
 * [`total_supply`](#total_supply)
```

Comment:
Shouldn't we remove `total_supply` from here and the corresponding section below as well?

Reply:
yep